src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml

```diff
@@ -182,6 +183,25 @@
                 <td>
                     <a class="review-name" asp-page="./Review" asp-route-id="@assembly.ReviewId">@assembly.DisplayName</a>
                 </td>
+                <td>
+                    @if (assembly.Revisions.Count > 0)
+                    {
+                        var lastRevision = assembly.Revisions.Last();
+                        if (lastRevision.Approvers.Count > 0)
+                        {
+                            <span>
+                                @foreach (var approver in lastRevision.Approvers)
+                                {
+                                    <label class="text-success">✔️@approver</label>
```

Comment:
Do we care about the list of people on the main page or just the fact that it was approved?

Reply:
Reason is to make it easier but not really add too much value. As mentioned in previous comment, Changing value to Approved or Pending is better option.